### PR TITLE
Add linz-lds-bde-schema-publish script

### LINKED ISSUE #175: Add linz-lds-bde-schema-publish script
Add linz-lds-bde-schema-publish script for adding tables to `all_lds` logical replication publication.

Similar as here: https://github.com/linz/linz-bde-schema/pull/149/

### PR DESCRIPTION
**Pull request by strk**
Creates an "lds" publication containing all tables in LDS
and BDE_EXT schemas.

Closes #175

### CODE REVIEW

File: scripts/linz-lds-bde-schema-publish.in [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+{
+cat << EOF
+DO \$PUBLICATION\$
+DECLARE
+    rec RECORD;
+BEGIN
+
+    IF NOT EXISTS ( SELECT 1 FROM pg_catalog.pg_publication
```
Comment (imincik) [inline scripts/linz-lds-bde-schema-publish.in]: * We want `all_bde_ext` publication for `bde_ext` schema and `all_lds` publication for `lds`schema

Comment (strk) [reply scripts/linz-lds-bde-schema-publish.in]: Ok for the split. For naming I liked the idea of publication name matching schema name, why do you want an "all_" prefix ?

Comment (imincik) [reply scripts/linz-lds-bde-schema-publish.in]: `all` is because we can have custom publications which will publish only subset of schema

Comment (strk) [reply scripts/linz-lds-bde-schema-publish.in]: custom publication can still be named differently, like "geodetic" or the like ?

Comment (imincik) [reply scripts/linz-lds-bde-schema-publish.in]: yes, can be. So let's come with some decision about that - if you keep your current names (I am not strictly against) I'll have to update diagrams and my code.

**Comment by strk** [comment]
Changing diagrams sounds painful, I'll leave that decision to you.

I'm not _strictly_ against "all_" prefix, just slightly confused
by why one would want an "all" prefix to something that's already
named after a schema and publishes all tables in that schema.

Think in general terms, this is a public repository.
How would you like to call a publication publishing all
tables that are created by a package called "linz-lds-bde-schema"
in schemas named `bde_ext` and `lds` ?

**Comment by imincik** [comment]
I think I lean more to `all_` prefix . Would you please change it everywhere in your code ?

**Comment by strk** [comment]
> I think I lean more to `all_` prefix . Would you please change it everywhere in your code ?

Done both here and in linz-bde-schema pr

**Pull request merged**